Formula/k3d.rb

```diff
@@ -1,33 +1,43 @@
 class K3d < Formula
   desc "Little helper to run Rancher Lab's k3s in Docker"
   homepage "https://github.com/rancher/k3d"
-  url "https://github.com/rancher/k3d/archive/v1.3.3.tar.gz"
-  sha256 "23cd26f1a1cf9feef25c2d0a731a0405991b71edc62d4207352c94ae4e34c5b8"
+  url "https://github.com/rancher/k3d/archive/v1.3.4.tar.gz"
+  sha256 "3d6c5d64795e4b459f236c391dd24e1ff08fcb2bf29e914509c8ddf8f699bfe7"
 
   bottle do
     cellar :any_skip_relocation
     sha256 "98fc996c777fe79b6ba0260cc7ef0e12c0961ee1462dbef12eddbeda8bb13fd3" => :catalina
     sha256 "3ad2b2ea2584c46837dcd958fa0b384bee15f193d85df3025c2b7d476429d6c6" => :mojave
     sha256 "3cb8ca78ded4bd85a41c4869ff6932a912368479d3c4d8ee73a7d77c26f589c5" => :high_sierra
+    sha256 "fb7377ca945eb5cefae081ccc02b7f51ae667b777af689ace480992993dee05a" => :x86_64_linux
```

Comment:
Version bumps should not touch the `bottle do` block.

Reply:
Not sure, what do do here then, since I didn't plan to only bump the version in the first place, but to add linux (it already existed in linuxbrew-core but not here) and update the installation and test steps (disable CGO and insert versions).

Reply:
Our continuous integration infrastructure will update the bottle block automatically. Please remove this changed line.

Reply:
DOne :+1: